README.md

```
@@ -1,53 +1,258 @@
+# 🚀 NucleusRV
 
-[![Join the chat at https://gitter.im/merledu/nucleusrv](https://badges.gitter.im/merledu/nucleusrv.svg)](https://gitter.im/merledu/nucleusrv?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+<p align="center">
+  <b>A 5-Stage Pipelined RV32I CPU written in Chisel</b><br>
+  Lightweight • Educational • Hackable
+</p>
 
-A chisel based riscv 5-stage pipelined cpu design, implementing 32-bit version of the ISA (incomplete).
+<p align="center">
+  <a href="https://github.com/merledu/nucleusrv/stargazers">
+    <img src="https://img.shields.io/github/stars/merledu/nucleusrv?style=for-the-badge" />
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)